src/modules/Dashboard/features/Managers/Managers.utils.tsx

```diff
@@ -18,30 +18,30 @@ export const getHeadCells = (id?: string): HeadCell[] => {
     {
       id: 'firstName',
       label: t('firstName'),
-      enableSort: true,
+      enableSort: sortableColumns?.includes('firstName') ?? true,
```

Comment:
I'm not 100% sure about the fallback value (`?? true`), both here and throughout the similar `getHeadCells` fns).

I get that it allows for this PR to be more "backwards compatible" in preserving the existing sort options in the event that this might be merged without the BE pull request (or otherwise address cases where `sortableColumns` would be undefined) — but this existing functionality is broken anyway, no? Would it make more sense to just default to `false`, and prevent the UI from exposing sort options that lack proper support?

To put it another way — I feel like correcting existing broken behaviour, even if just by omission, is better than preserving it.

Reply:
Yes, you're probably right! I was on the fence about it but you've convinced me, will update.